src/SFA.DAS.VacancyServices.Functions/SchedulerFunctions/SchedulerFunctionConstants.cs

```diff
@@ -0,0 +1,20 @@
+namespace SFA.DAS.VacancyServices.Functions.SchedulerFunctions
+{
+    public static class SchedulerFunctionConstants
+    {
+        public const string RecruitV1StorageConnectionStringKey = "RecruitV1StorageConnectionString";
```

Comment:
the storage connection string isn't going to be able to be a constant, it will be configured in the settings?

Reply:
yes and this is key to identify the config setting

Reply:
ah misread the value, could we not just inject the config into the queue service ?

Reply:
Do you mean use options pattern? The service has dependency on the IConfig... it still needs to know which setting to read, hence this constant.